src/args.ts

```diff
@@ -9,9 +10,13 @@ const singleStatRecordSchema = z.partialRecord(
   z.int().min(1)
 )
 
-export const argsToPartialStats = (args: string[]): Partial<Record<StatLiteral, number>> =>
-  args.map(arg => {
+export const parseArgs = (args: string[]) => {
```

Comment:
Tähän vois lisää tyypityksen ni on selkeempi öögata mitä tuolta tulee ilman että hoveroi IDEllä. Tää on aika potentiaalinen paikka johon tulee vielä lisää kenttiä niin siksikin.

Lisäksi kun tossa alkaa olla kampetta, muutin formatoinnin "prettier-tyyliseksi".
```suggestion
export const parseArgs = (
  args: string[]
): { stats: Partial<Record<StatLiteral, number>>, shouldSave: boolean } => {
```